### PR TITLE
Update issue type

### PR DESCRIPTION
**Pull request by dmh43**
Thank you so much @ahungry for putting in the time to fix the REST API issues!

I've added the ability to update issue types. It's done in the same way as issue priority.

### CODE REVIEW

File: org-jira.el [hunk 1]
```diff
@@ -1020,6 +1020,12 @@ See`org-jira-get-issue-list'"
    (org-jira-refresh-issue)))
 
 
+(defun get-id-name-alist (name ids-to-names)
```
Comment (ahungry) [inline org-jira.el]: Good abstraction, but could you name it with the org-jira prefix?  So `(defun org-jira-get-id-name-alist` (also add a docstring in format that makes flycheck happy, something like `"Returns an alist containing id and name keys, generated from NAME and IDS-TO-NAMES."` (that could probably be worded better)

Comment (dmh43) [inline org-jira.el]: yes of course!

Comment (dmh43) [inline org-jira.el]: done

**Comment by ahungry** [comment]
Thanks, merged!

**Pull request merged**